shared/actions/favorite.js

```diff
@@ -116,145 +163,102 @@ const folderToState = (folders: Array<FolderWithMeta>, username: string = ''): F
   }
 }
 
-// If the notify data has changed, show a popup
-let previousNotifyState = null
-
-const injectMeta = type => f => { f.meta = type }
-
-const jsonToFolders = (json: Object, myKID: any) => {
-  const folderSets = [json.favorites, json.ignored, json.new]
-  folderSets.forEach(folders => {
-    folders.forEach(folder => {
-      folder.waitingForParticipantUnlock = []
-      folder.youCanUnlock = []
+function _getFavoritesRPCToFolders (txt: string, username: string = '', loggedIn: boolean): Array<FolderWithMeta> {
```

Comment:
🐑 I think saga's should be together near the bottom of the file, is there a reason